com.gopeed.Gopeed.yaml

```diff
@@ -0,0 +1,55 @@
+app-id: com.gopeed.Gopeed
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+rename-desktop-file: gopeed.desktop
+command: gopeed
+finish-args:
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--share=network'
+  - '--device=dri'
+  - '--talk-name=org.kde.StatusNotifierWatcher'
+  - '--talk-name=org.freedesktop.Notifications'
+  # https://github.com/miguelpruivo/flutter_file_picker/pull/1275
+  - '--filesystem=home'
```

Comment:
Can this be xdg-download only? Seems to be the default location

Reply:
I've updated filesystem